web/server/entrypoint_linux.go

```diff
@@ -1,17 +1,23 @@
-//go:build cgo && linux && !android
+//go:build cgo && linux && !arm
```

Comment:
@abe-winter I'm leaving the 32-bit alone for now so I'm not exactly sure why the 32-bit tests are failing but it doesn't look like it has anything to do with these changes, especially considering main has been acting janky. I'll keep looking into though until I get a definitive answer.

Reply:
(32 bit passing now, resolving this)